src/log.js

```diff
@@ -32,7 +31,11 @@ module.exports = {
 		read(options, callback);
 	},
 
-	getHumanDate(ts) {
-		return moment(ts).format("YYYY-MM-DD HH:mm:ss");
+	getHumanDate(date) {
+		if (!date) {
+			date = new Date();
+		}
+
+		return date.toISOString().split(".")[0] + "Z";
```

Comment:
This drops `.000` milliseconds from the string. Unless we want to keep it?

Reply:
I would vote to keep them. Added precision can't hurt, esp. if it makes the code simpler.
Is it always `.000` or some/all servers send precision to milliseconds? I don't know, even if it's always `.000`, it might not hurt to keep them, future-wise.

Reply:
Note to self, you mentioned on IRC that time is already UTC and not server time. In https://github.com/thelounge/thelounge/pull/2506 we mentioned server time, so I need to edit the upgrade guide accordingly (I also prefer storing logs in UTC, personally, so that's good).

For the record:

```
$ date
Thu Sep 20 21:18:12 EDT 2018
$ node
> new Date().toISOString()
'2018-09-21T01:18:18.274Z'
```

🤷‍♂️

Reply:
@McInkay What do you think?

Reply:
I guess the only real reason to remove the milliseconds is if we assume there are some people using scripts to read their logs. Which I assume there probably aren't many, if any. And we are already changing the format, aren't we?